docs/shared/guides/encapsulated-nx-and-the-wrapper.md

```diff
@@ -0,0 +1,53 @@
+# Encapsulated Nx
+
+`nx init` accepts a flag called `--encapsulated`. When this flag has been passed, Nx manages its own installation separately from any dependencies that are local to your repository. As such, `nx` and any plugins will not be listed in a root level package.json, and also will not be in a root level node_modules folder. In fact, these are not required for an encapsulated Nx repository.
+
+To manage the local Nx installation, and invoke Nx, a total of 4 files are created.
+
+- [.nx/nxw.js](#nx-wrapper)
+- [nx](#nx-and-nxbat)
+- [nx.bat](#nx-and-nxbat)
+- [nx.json](#nx-json)
+
+{% callout type="note" title="Available since Nx v15.7.0" %}
+
+Support for `--encapsulated` was added in Nx v15.7.0. To ensure that it is available, specify the version of nx when running your command so that `npx` doesn't accept an older version that is in the cache. (e.g. `npx nx@15.7.0 init --encapsulated` or `npx nx@latest init --encapsulated`)
```

Comment:
```suggestion
Support for `--encapsulated` was added in Nx v15.8.0. To ensure that it is available, specify the version of nx when running your command so that `npx` doesn't accept an older version that is in the cache. (e.g. `npx nx@latest init --encapsulated` or `npx nx@15.8.0 init --encapsulated`)
```